### ISSUE TITLE
Generate image sizes on the fly

### ISSUE THREAD

**Issue by trendschau** [typemill/typemill#108]
For themes the developer wants to create image sizes on the fly. Create a helper class that accepts url of image and desired sizes. It will check if the image with sizename like image-500x333.png exists, if not, it will generate it.

**Comment by trendschau** [comment]
Done. With next release 1.3.7 you can do the following in your twig templates: 

Use standard live image like today:

```
<img src="{{ metatabs.meta.heroimage }}" />
```

Resize image: 

```
<img src="{{ assets.image(metatabs.meta.heroimage).resize(800,500).src() }}" />
```

Grayscale image: 

```
<img src="{{ assets.image(metatabs.meta.heroimage).grayscale().src() }}" />
```

Resize image and then grayscale: 

```
<img src="{{ assets.image(metatabs.meta.heroimage).resize(800,500).grayscale().src() }}" />
```

**Comment by trendschau** [comment]
published today with version 1.3.7
